api/pyproject.toml

```diff
@@ -15,7 +15,7 @@ dependencies = [
   "aws-requests-auth >=0.4.3, <0.5",
   "deepdiff >=6.7.1, <7",
   "django >=4.2.11, <5",
-  "django-asgi-lifespan >=0.2.0, <0.3",
+  "django-asgi-lifespan >=0.3.1, <0.4",
```

Comment:
I expected Renovate to update `pdm.lock` within the range of `pyproject.toml`. But it seems like Renovate does not do that but rather just updates `pyproject.toml` to the latest version.

Reply:
Renovate doesn't have the ability to update specific lockfiles, just the explicit dependencies list (example here for browsers list: https://github.com/renovatebot/renovate/issues/8615#issuecomment-778139572).